### PR TITLE
Add Maistest Subnet

### PR DESCRIPTION
**Pull request by steliyannedev**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by steliyannedev** [comment]
@ligi formatted as requested.

**Comment by steliyannedev** [comment]
@ligi hey is there something else I should do?

File: _data/chains/eip155-43214913.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "infoURL": "",
+  "shortName": "mais",
+  "chainId": 43214913,
+  "networkId": 43214913,
+  "explorers": [
+    {
+      "name": "maistesntet",
+      "url": "http://174.138.9.169:3006/?network=maistesntet",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-43214913.json:21]:
```suggestion
      "standard": "none"
```

**Pull request merged**